index.html

```diff
@@ -4,464 +4,524 @@
+                  <div class="right-side__first-circle">
+                    <img src="./src/icons/avatar-7.svg" alt="first avatar" />
+                  </div>
+                  <div class="right-side__second-circle">
+                    <img src="./src/icons/avatar-6.svg" alt="second avatar" />
                   </div>
-                  <div class="cards-content__right-side">
-                    <div class="right-side__first-circle">
-                      <img src="./src/icons/avatar-7.svg" alt="first avatar" />
-                    </div>
-                    <div class="right-side__second-circle">
-                      <img src="./src/icons/avatar-6.svg" alt="second avatar" />
-                    </div>
-                    <div class="right-side__third-circle">
-                      <p class="avatar-text">5+</p>
-                      <img src="./src/icons/avatar-3.svg" alt="third avatar" />
-                    </div>
+                  <div class="right-side__third-circle">
+                    <p class="avatar-text">5+</p>
+                    <img src="./src/icons/avatar-3.svg" alt="third avatar" />
                   </div>
                 </div>
+              </div>
 
-                <!-- Cards content middle -->
-                <div class="cards-content__middle">
-                  <h2 class="cards-content__middle_title tltle">
-                    Project Manager
-                  </h2>
-                  <p class="cards-content__middle_subtitle">
-                    Full Time - Singapore
-                  </p>
-                </div>
+              <!-- card content middle -->
+              <div class="card-content__middle">
+                <p class="card-content__middle_title black-txt">
+                  Project Manager
+                </p>
+                <p class="card-content__middle_subtitle">
+                  Full Time - Singapore
+                </p>
+              </div>
 
-                <!-- Cards content bottom -->
-                <div class="cards-content__bottom">
-                  <a class="cards-content__button grey-btn" href="#">
-                    <p class="cards-content__button-text">Remote</p>
-                  </a>
-                  <a class="cards-content__button grey-btn" href="#">
-                    <p class="cards-content__button-text">Anywhere</p>
-                  </a>
-                  <p class="cards-content__button-text black-txt">
-                    $10k - $25k
-                  </p>
-                </div>
+              <!-- card content bottom -->
+              <div class="card-content__bottom">
+                <span class="card-content__button grey-btn" href="#">
+                  Remote
+                </span>
+                <span class="card-content__button grey-btn" href="#">
+                  Anywhere
+                </span>
+                <p class="card-content__button-text black-txt">$10k - $25k</p>
```

Comment:
```suggestion
                <p class="button__text black-txt">$10k - $25k</p>
```
Please, separate elements from blocks

Reply:
Separated